dev/com.ibm.ws.jpa.container.v31/bnd.bnd

```diff
@@ -0,0 +1,41 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
```

Comment:
Since Jakarta EE 10 requires Java 11 as a minimum, this project should probably specify:

```
javac.source: 11
javac.target: 11
```